docs/experimental/swap.md

```diff
@@ -0,0 +1,175 @@
+TRAC[2020-09-26T18:31:30+01:00] crediting peer f1e2872581de18bdc68060dc8edd3aa96368eb341e915aba86b450486b105a47 with price 110, new balance is -85130
+TRAC[2020-09-26T18:31:30+01:00] pusher pushed chunk f503b2feae7c0ced9ca786fa125e9798a3ce1ae1ac20c8c6ead6ecb786e8db23
+TRAC[2020-09-26T18:31:30+01:00] crediting peer f1e2872581de18bdc68060dc8edd3aa96368eb341e915aba86b450486b105a47 with price 150, new balance is -85280
+TRAC[2020-09-26T18:31:30+01:00] pusher pushed chunk b68b01815a2d9b5ab59a0cf6452151ccec871a5195eb5cec96210c874a42d690
+TRAC[2020-09-26T18:31:30+01:00] crediting peer e00460ced3c509dfd72b6ce915c764b13669e65f95b3ba84dcb7d4b6d18a0b11 with price 150, new balance is -82920
+TRAC[2020-09-26T18:31:30+01:00] crediting peer 6a04f1f872da2d32a4597fad2323e8c5f4a00cb441c2bba2fa87f771dd358572 with price 130, new balance is -38230
+TRAC[2020-09-26T18:31:30+01:00] pusher pushed chunk a4b548a1426b47798c6d011d95ba2626a740aea49a26a1435210a307240c260e
+TRAC[2020-09-26T18:31:30+01:00] pusher pushed chunk 7243a785fa09e573f8db4f50ecfe2a1a67fe0d03fc94914e26d5cfec549331dc
+...
+```
+
+We also have a rich set of features to be able to query the current accounting state of your node. For example, you may query your node's current balance by send a POST request to the balances endpoint.
+
+```sh
+curl localhost:6060/chequebook/balance | jq
+```
+
+```json
+{
+  "totalBalance": 10000000,
+  "availableBalance": 9640360
+}
+```
+
+It is also possible to examine per-peer balances.
+
+```sh
+curl localhost:6062/balances | jq
+```
+
+```json
+{
+  "balances": [
+    //...
+    {
+      "peer": "d0bf001e05014fa036af97f3d226bee253d2b147f540b6c2210947e5b7b409af",
+      "balance": -85420
+    },
+    {
+      "peer": "f1e2872581de18bdc68060dc8edd3aa96368eb341e915aba86b450486b105a47",
+      "balance": -75990
+    }
+    //...
+  ]
+}
+```
+
+In Swarm, these per-peer balances simply represent trustful agreements between nodes. Tokens only actually change hands when a node settles a cheque. This can either be triggered manually or when a certain threshold is reached with a peer. In this case, a settlement takes place. You may view these using the settlements endpoint.
+
+```sh
+curl localhost:6062/settlements | jq
+```
+
+```json
+{
+  "totalreceived": 0,
+  "totalsent": 359640,
+  "settlements": [
+    // ...
+    {
+      "peer": "d0bf001e05014fa036af97f3d226bee253d2b147f540b6c2210947e5b7b409af",
+      "received": 0,
+      "sent": 90070
+    },
+    {
+      "peer": "f1e2872581de18bdc68060dc8edd3aa96368eb341e915aba86b450486b105a47",
+      "received": 89890,
+      "sent": 0
+    }
+    // ...
+  ]
+}
+```
+
+As our node's participation in the network increases, we will begin to see more and more of these balances arriving. In the case that we have received a settlement from another peer, we can ask our node to perform the relevant transactions on the blockchain, and cash our earnings out.
+
```

Comment:
We should at least add something about `/chequebook/cheque`/`/chequebook/cheque/<peer>`. 

`/chequebook/deposit` and `/chequebook/withdraw` could also be added but are not as important.